extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/ObjectQueryService.java

```diff
@@ -220,6 +220,9 @@ public String generateQuery(String filter,
                         logger.warn("Given a referral, but referrals are disabled. Error was: {}", e.getMessage());
                         logger.debug("Got a referral, but configured to not follow them.", e);
                     }
+                } catch (LDAPException e) {
```

Comment:
Please don't cuddle the `catch`. We do that for `do`/`while`, but that's really it. See [the rough guidelines](http://guacamole.apache.org/guac-style/#braces) and elsewhere:

https://github.com/apache/guacamole-client/blob/b3f3b18f0bbd063b4f9ce8ce50329ac3f1e47148/extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/ObjectQueryService.java#L208-L213

Reply:
Fixed.  What do you think about adding a link to those coding guidelines in the CONTRIBUTING document?